Add the Create WebP Image package

TomasBarry opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request
and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass.
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can request a review from @packagecontrol-bot
to manually trigger an automated review
if you don't need to push a new commit.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"`
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.

You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists,
why you believe it is different and needed
below this line. -->

A redo of https://github.com/wbond/package_control_channel/pull/7867 since that one went stale.

The necessary updates have been made and I have verified that this package works.

Ultra-Instinct-05 commented:
A couple points, but otherwise seems fine.

1. https://github.com/TomasBarry/Sublime-create-webp-image/blob/master/.DS_Store - I suppose this is something MacOS specific and can be gitignored ?

2. https://github.com/TomasBarry/Sublime-create-webp-image/blob/master/CreateWebpImage.sublime-settings#L9 - 99% of the times, settings in Sublime Text Packages are snake_case by default. But whatever suits you.

3. https://github.com/TomasBarry/Sublime-create-webp-image/blob/master/CreateWebpImage.py#L89 - You use the `input` method only when you are implementing an `InputHandler`. Otherwise, it's not required.

TomasBarry commented:
Thanks for the approval @Ultra-Instinct-05. What needs to happen to get this merged in?

braver commented on repository/c.json:
```diff
@@ -3937,6 +3937,19 @@
 			]
 		},
+		{
+			"author": ["Tomas Barry"],
```
I didn't know we supported arrays here, also doesn't seem necessary in this case.

TomasBarry replied:
Addressed.

braver commented on repository/c.json:
```diff
@@ -3937,6 +3937,19 @@
+		{
+			"author": ["Tomas Barry"],
+			"name": "Create WebP Image",
+			"description": "Convert images to WebP from Sublime Text",
```
You can put this in the GitHub repo description instead for ease of maintenance.

TomasBarry replied:
Addressed.

braver commented:
I think you'd want to be more clear about the prerequisite. From experience I guess I need to install that utility in a way that puts cwebp in my $PATH, but for many users that's a few hurdles to many.

TomasBarry commented:
@braver, I've addressed your comments and have included the latest changes as part of the release (a lapse on my part). The usage instructions have been updated to more explicitly reference the official documentation so that I always point to the latest instructions instead of potentially having stale instructions in my README.

Is there anything else I need to cover?

braver commented:
Nope, all good 👍🏻
